M2/Macaulay2/m2/ringmap.m2

```diff
@@ -193,9 +193,7 @@ RingMap MutableMatrix := MutableMatrix => (p,m) -> (
      then error "expected source of ring map to be the same as ring of matrix";
      map(S,rawRingMapEval(raw p, raw m)))
 
-RingMap Vector := Vector => (p,m) -> (
-     f := p new Matrix from m;
-     new target f from f)
+RingMap Vector := Vector => (p,m) -> vector p new Matrix from m
```

Comment:
Very minor, but I think just `vector p m#0` works here.

Kinda strange to have `new Matrix from Vector` when `matrix Vector` does the same (though perhaps `matrix(Vector, Degree => d)` should use the option).

Reply:
Yeah, I was just blinding replacing `new target f from` with `vector` without looking at the stuff afterwards.  This definitely could be improved.